### ISSUE TITLE
[CAIA a11y Research - PILOT]: Disability Benefits Experience Team 1 (DBEX 1), 526ez (Desktop) (Nov. 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#97152]
> [!IMPORTANT]
> - **TICKET STATUS:** `Being Re-Scheduled`
> - **PILOT DATE:** ~11/21/2024~ to ~11/25/2024 1:30 - 3 p.m. EST~ to `New Time - TBD - 11/27 at 1:30 p.m. ET`
> - **DEVICE:** Desktop
> - **TYPES OF ASSISTIVE TECH:** Windows + NVDA
> - **ENVIRONMENT:** [Prototype in Staging](https://staging.va.gov/disability/file-disability-claim-form-21-526ez/introduction)
> - **INTERNAL NOTES DOC:** [Google Doc for Pilot Notes](https://docs.google.com/document/d/1gPjUnWUJrNE_-jiCU_5sjgAJMS93hlmbJ8vrcffU4o8/edit?tab=t.0#heading=h.7b53y4dqihzk)

# CAIA A11y Support
- This is a child ticket of the research ticket [96810](https://github.com/department-of-veterans-affairs/va.gov-team/issues/96810) - created to track work, completed from this **Desktop Pilot** session.
    - This ticket details the resources needed to complete the **Desktop Pilot**.
- The **Mobile Pilot** has been `completed`: 
    - [ ] Review feedback from the [**mobile** pilot session on 11/21/2024.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/97151#issuecomment-2499099484)
    - [ ] Review the full notes in the [Google Doc from the **mobile** pilot session on 11/21/224 ](https://docs.google.com/document/d/1QFs5mDiVO1kgoOi4GpOgnDvkktznmnjCBDn8dza016Q/edit?tab=t.0#heading=h.7b53y4dqihzk)for reference, if needed. 

> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

- **Availability**
    - @SarahKay8 (pilot)
    - @EvanAtCoforma (co-pilot)
    - @sara-amanda OOO 11/26-29

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name: Disability Benefits Experience Team 1
- OCTO-DE product owner: Emily Theis
- Product manager: Jared Pace
- Researchers: Maya Carroll, Evelyn Hilbrich Davis, Lauren Dawson
- FE engineer: Christine Cereca, Tracy Tran
- Product/team Slack channel: benefits-disability

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
- **Date:** ~11/21/2024 3-4:30 p.m. EST (hard stop)~ changed to ~11/25/2024~ changed to `TBD`
- [**Prototype in Staging**](https://staging.va.gov/disability/file-disability-claim-form-21-526ez/introduction)
- **Location**: Staging
- **Available after 11/18/2024**
    - [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/research/2024-11-Toxic%20Exposure/Non-AT%20Research%20Plan.md)
    - [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/research/2024-11-Toxic%20Exposure/AT-%20Conversation%20Guide.md)

</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11y** 11/21/2024
- [x] **Re-schedule CAIA a11y** 11/25/2024
- [x] **Share link to [Assistive Tech Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with team, if they haven't reviewed it yet**
- [x] **Determine date and time for pilot:** 11/21 (SK and/or Jamie) or 11/22 (Jamie only) 11/21 works - hard stop at 4:30 p.m. EST
- [x] **Calendar Hold** pending invite
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled
- [x] **Review research** [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/research/2024-11-Toxic%20Exposure/AT-%20Conversation%20Guide.md) and provide feedback in comments.
- [x] [Request AT device and tech to be used in sessions](LINK)
- [x] [Receive AT device and tech from Team in research ticket](LINK) to be used in sessions to prep for pilot
- [x] Update the Research Tracker
- [x] **Review prototype** in [Staging](https://staging.va.gov/disability/file-disability-claim-form-21-526ez/introduction) (users 54 or 229)
- [x] **Re-schedule CAIA a11y** 11/27/2024? `TBD`
- [x] **Run pilot** with VFS Researcher
- [x] **Optional:** [Note Taking](https://docs.google.com/document/d/1gPjUnWUJrNE_-jiCU_5sjgAJMS93hlmbJ8vrcffU4o8/edit?usp=sharing)
```

**Comment by sara-amanda** [comment]
## Scheduling Pilot(s)

Evelyn Hilbrich Davis

> Hello, I would like to schedule our 2 pilot sessions for mobile/desktop. We are planning the sessions for 90 minutes.
> Friday 11/22, 11:30-3pm EST
> Monday 11/25 1-3pm EST
> Tuesday 11/26 1-3pm EST.
> [@sitewide-caia-accessibility](https://dsva.slack.com/admin/user_groups) :thank_you:

cc: @coforma-jamie

**Comment by EvanAtCoforma** [comment]
# Accessibility Feedback for DBEX 526ez Pilot

## Important Messages

- For important messages such as information alerts, success alerts and confirmation alerts, ensure these messages are conveyed to the user upon page load using `aria-live="polite"`. This will ensure these messages are not missed by screen reader users, as users would need to search for this information if not explicitly mentioned.

## More Information in Date Fields

- While these date fields are VA-approved components, we still want to provide some feedback to bring to your attention. For blind users relying their screen reader to provide all the information needed to perform a task, the format expectation for the date (“Month and Year”) might not be clear enough.

### Recommendation

 - Add an accessible hint to each field using aria-describedby to clarify what is required:
    - `<select id="month" aria-describedby="month-hint"></select>`
       `<span id="month-hint">Select a month</span>`
      `<input id="year" aria-describedby="year-hint" />`
      `<span id="year-hint">Enter a year in YYYY format</span>`

**Comment by sara-amanda** [comment]
> [!NOTE]
> ## CAIA INTERNAL
> - [x] This ticket is linked to its parent research ticket.
> - [x] This work has been completed
> - [x] Closing ticket
> - [x] Adding to tracker for CAIA Sprint 140
